assembl/static2/js/app/graphql/Post.graphql

```diff
@@ -28,7 +28,7 @@ fragment Post on Post {
   creator {
     ...AgentProfileInfo
   }
-  # parentId
+  parentId
```

Comment:
no need, you should already have parentId in props.parentId from the IdeaWithPosts query

Reply:
Because this is happening on the post component itself, we observed that parentId is not available in the constructor. Why was this commented in the first place?

Reply:
Why do you want it in the constructor, to set an initial state? There is no need for a state here. It is commented to not fetch it again, we already get this data with the IdeaWithPosts query. See http://dev.apollodata.com/react/query-splitting.html for the concept of query splitting. The IdeaWithPosts and PostQuery should not overlap.

Reply:
Ok c'est réglé, merci des commentaires Vincent!